slowzones.py

```diff
@@ -42,6 +47,9 @@
     access_token=MASTODON_ACCESS_TOKEN,
 )
 
+bluesky_client = atproto.Client(base_url=ATP_PDS_HOST)
+bluesky_client.login(login=ATP_AUTH_HANDLE, password=ATP_AUTH_PASSWORD)
```

Comment:
This seems to be what's failing the dry run: https://github.com/transitmatters/mbta-slow-zone-bot/actions/runs/11939535356/job/33280264410?pr=53

Maybe this client login should be moved under L126 so it's captured by our existing exception?

If so we should follow suit with the other domains, but that can be a separate PR.